### PR TITLE
Homework Submit Wk 1

### PR DESCRIPTION
**Pull request by AlissaHeidi**

### CODE REVIEW

File: Week_01_Basics/Submissions/Portfolio.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	<nav>
+		<a href="About Me.html">ABOUT ME</a> // <a href="Portfolio.html">PORTFOLIO</a>
+	</nav>
+
+	<hr> </hr>
+	<h3> Relaxr Web Design </h3>
+	<p> Designed a fully functional website for Relaxr. </p>
+	<div> <img src="./relaxr_deliverable.png"> </div>
+
+	<hr> </hr>
```
Comment (yangli-io) [inline Week_01_Basics/Submissions/Portfolio.html:16]: You didnt close your `<body>`

File: Week_01_Basics/Submissions/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
+main {
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 14px;
+}
+
+h1 {
+	font-family: "Lucida Grande", sans-serif;
```
Comment (yangli-io) [inline Week_01_Basics/Submissions/stylesheet.css:8]: inconsistent indentation here

File: Week_01_Basics/Submissions/About Me.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<head>
```
Comment (yangli-io) [inline Week_01_Basics/Submissions/About Me.html:1]: you're missing the `<!doctype html>` amd `<html>` tags

**Comment by yangli-io** [comment]
Great work, just missing a `<title>`

**Comment by AlissaHeidi** [comment]
Thanks Yang,

I'm sorry but I haven't been able to do the homework for this week, thanks for getting this to me though, see you tonight!

**Pull request closed without merging**